sass/_mixins.scss

```diff
@@ -54,3 +53,11 @@
   -o-text-shadow: 5px 5px 2px black;
   text-shadow: 5px 5px 2px black;
 }
+
+//Styles for hover span with <a> tag
+@mixin hover_link {
+  text-decoration: none;
+  font-weight: bold;
+  color: $primary_color;
+  transition: all 0.3s ease;
+}
```

Comment:
Awesome. GG! :)

Reply:
GG????? Thanks :)

Reply:
GG or GJ stand for Good game! / Good Job! :)
The first one is usually used in the gaming area/teams.